Publish JS packages action does not seem to be working

For some reason the Publish JS Packages action is failing: https://github.com/deephaven/deephaven-plugins/actions/runs/9863612383

```
Found 1 package to publish:
 - @deephaven/js-plugin-ui => 0.17.0

lerna info auto-confirmed 
lerna info publish Publishing packages to npm...
lerna ERR! E404 Not found
Error: Process completed with exit code 1.
```
Also, after manually publishing, the `README.md` was not getting published with the package:
![image](https://github.com/deephaven/deephaven-plugins/assets/4505624/425a0db9-40f5-4af6-9b79-655603ebabaa)
![image](https://github.com/deephaven/deephaven-plugins/assets/4505624/6505f1e4-f029-4efb-a80c-4a25c7de39d5)

--- a/.github/workflows/publish-alpha.yml
+++ b/.github/workflows/publish-alpha.yml
@@ -16,6 +16,7 @@ jobs:
     permissions:
       contents: read
       packages: write
+      id-token: write
     steps:
       - uses: actions/checkout@v3
         with:

--- a/.github/workflows/publish-packages.yml
+++ b/.github/workflows/publish-packages.yml
@@ -9,6 +9,7 @@ jobs:
     permissions:
       contents: read
       packages: write
+      id-token: write
     steps:
       - name: Checkout
         uses: actions/checkout@v3

--- a/.npmrc
+++ b/.npmrc
@@ -1,0 +1,1 @@
+provenance=true

